clerk/clerk_options.go

```diff
@@ -0,0 +1,37 @@
+package clerk
```

Comment:
Maybe rename the file name to `client_options.go`, to match the actual type.

Reply:
Actually renamed this from `ClientOption` to `ClerkOption`, so there is no confusion if this options are for the clerk client or the client resource